### PR TITLE
feat(icons): update to new globe icon and add new globe tinycon

### PR DESCRIPTION
**Pull request by cweeii**
Old:
image

New:
image

New Tinycon:
image

### CODE REVIEW

**Comment by boxcla** [comment]
Verified that @cweeii has signed the CLA. Thanks for the pull request!

File: src/icons/general/IconGlobe.js [hunk 1]
```diff
@@ -12,18 +13,17 @@ type Props = {
...
+const IconGlobe = ({ className = '', color = bdlNeutral03, height = 16, title, width = 16 }: Props) => (
     <AccessibleSVG
         className={`icon-globe ${className}`}
         height={height}
         title={title}
         viewBox="0 0 16 16"
         width={width}
     >
-        <circle className="fill-color" cx="8" cy="8" fill={color} r="8" />
         <path
```
Comment (j3tan) [inline src/icons/general/IconGlobe.js]: `fill-color` class

File: src/icons/general/IconGlobe.js [hunk 2]
```diff
@@ -12,18 +13,17 @@ type Props = {
     width?: number,
 };
 
-const IconGlobe = ({ className = '', color = '#000', height = 16, title, width = 16 }: Props) => (
+const IconGlobe = ({ className = '', color = bdlNeutral03, height = 16, title, width = 16 }: Props) => (
```
Comment (priyajeet) [inline src/icons/general/IconGlobe.js]: 32x32 please

**Comment by priyajeet** [comment]
@cweeii can this be made into the two-toned icons? See two toned directory

File: src/icons/general/IconGlobe.js (hunk 1 above)
Comment (cweeii) [reply src/icons/general/IconGlobe.js]: done

File: src/icons/general/IconGlobe.js (hunk 2 above)
Comment (cweeii) [reply src/icons/general/IconGlobe.js]: @priyajeet I've made it 20x20 as it's mainly used as a badgeable icon

**Comment by cweeii** [comment]
@priyajeet I don't see a need for background/foreground color for the icons I'm editing/adding, so do they need to be two-toned?

File: src/icons/general/__tests__/IconGlobeTinycon-test.js [hunk 3]
```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+
+import { white } from '../../../styles/variables';
+
+import IconGlobeTinycon from '../IconGlobeTinycon';
```
Comment (booc0mtaco) [inline src/icons/general/__tests__/IconGlobeTinycon-test.js]: We don't have any naming scheme for the different usages of icons. @priyajeet is there any scheme that might make sense from your perspective? I was thinking a usage prefix like `Tinycon` which also implies associated default sizing, etc.

**Comment by priyajeet** [comment]
@booc0mtaco @tjgupta I think we should make all svg icons same viewport 32x32 irrespective of their use. If it needs to be a 20x20 icon for badges, then the width/height should be modified. Something like this https://codesandbox.io/s/peaceful-poitras-rc47d

**Pull request merged**